lib/ansible/modules/uri.py

```diff
@@ -677,7 +677,9 @@ def main():
     resp, content, dest = uri(module, url, dest, body, body_format, method,
                               dict_headers, socket_timeout)
     resp['elapsed'] = (datetime.datetime.utcnow() - start).seconds
-    resp['status'] = int(resp['status'])
+    # Note for file:// URL's a valid response is None, which we
+    # convert to 200 here.
+    resp['status'] = int(resp['status']) if resp['status'] is not None else 200
```

Comment:
Are there cases is which the API will return `None` for regular URLs? It seems we should restrict this `None` to `200` conversion to just `file://` URLs.